pkgs/applications/audio/flacon/default.nix

```diff
@@ -21,12 +19,23 @@ stdenv.mkDerivation rec {
   postInstall = ''
     wrapProgram $out/bin/flacon \
       --suffix XDG_DATA_DIRS : "${gtk3}/share/gsettings-schemas/${gtk3.name}" \
-      --prefix PATH : "${lib.makeBinPath [ shntool flac opusTools vorbis-tools
-     mp3gain lame wavpack vorbisgain ]}"
+      --prefix PATH : "${
+        lib.makeBinPath [
+          shntool
+          flac
+          opusTools
+          vorbis-tools
+          mp3gain
+          lame
+          wavpack
+          vorbisgain
+        ]
```

Comment:
Would make sense to move this to a separate attribute, like:
```
stdenv.mkDerivation {
  bin_path = lib.makeBinPath [
    # ...
  ];
  postInstall = ''
    wrapProgram $out/bin/flacon \
      --prefix PATH : "$bin_path"
  '';
}
```
Notice that this uses `$bin_path` the environment variable, not the Nix string interpolation `${bin_path}`. This allows for overriding like `flacon.overrideAttrs (_: { bin_path = /* ... */; })` to work.